plugins/org.wso2.developerstudio.visualdatamapper.diagram/src/org/wso2/developerstudio/datamapper/diagram/xslt/config/XSLTGeneratorConstants.java

```diff
@@ -0,0 +1,144 @@
+    public static final String NAME = "name";
+    public static final String SELECT = "select";
+    public static final String TEST = "test";
+
+    // xsl tag names
+    public static final String XSL_VALUE_OF = "xsl:value-of";
+    public static final String XSL_PARAM = "xsl:param";
+    public static final String XSL_IF = "xsl:if";
+    public static final String XSL_VARIABLE = "xsl:variable";
+    public static final String XSL_FOR_EACH = "xsl:for-each";
+    public static final String XSL_TEMPLATE = "xsl:template";
+    public static final String XSL_ATTRIBUTE = "xsl:attribute";
+
+    // node types
+    public static final String AT_NODE = "/@node";
+    public static final String AT_OPERATORS = "//@operators";
+    public static final String ATTRIBUTES_INITIALS = "@";
+
+    public static final String INCOMING_LINK = "incomingLink";
+    public static final String OUT_NODE = "outNode";
+    public static final String IN_NODE = "inNode";
+    public static final String LEVEL = "level";
+    public static final String PROPERTIES_LOWER_CASE = "properties";
+    public static final String KEY = "key";
+    public static final String NODE = "node";
+    public static final String VALUE = "value";
+    public static final String LEFT_CONNECTORS = "leftConnectors";
+    public static final String LEFT_CONTAINER = "leftContainer";
+    public static final String BASIC_CONTAINER = "basicContainer";
+
+    public static final String XMLNS_XSL = "xmlns:xsl";
+    public static final String XMLNS_XS = "xmlns:xs";
+    public static final String VERSION = "version";
+    public static final String XMLNS_OWN = "xmlns:own";
+    public static final String XSL_STYLESHEET = "xsl:stylesheet";
+    public static final String MATCH_LOWER_CASE = "match";
+    public static final String XSL_FUNCTION = "xsl:function";
+    public static final String OWN_SET_PRECISION = "own:setPrecision";
+    public static final String RESULT_STRING = "resultString";
+    public static final String EMPTY_STRING = "";
+    public static final String DOT_SYMBOL = "\\.";
+    public static final String SLASH = "/";
+    public static final String SCOPE = "scope";
+    public static final String DEFAULT_SCOPE = "DEFAULT";
+    public static final String DEFAULT_VALUE = "defaultValue";
+    public static final String DEFAULT_NAME = "defaultName";
+    public static final String CUSTOM_FUNCTION = "CUSTOM_FUNCTION";
+    public static final String FUNCTION_DEFINITION = "functionDefinition";
+    public static final String XSL_NAMESPACE_URI = "http://www.w3.org/1999/XSL/Transform";
+    public static final String XS_NAMESPACE_URI = "http://www.w3.org/2001/XMLSchema";
+    public static final String XSLT_VERSION = "2.0";
+    public static final String XSLT_FUNCTION_DECLARE_URI = "http://ownfunction.org";
+    public static final String RUN_TIME_PROPERTIES = "xmlns:runTimeProperties";
+    public static final String NOT_XSLT_COMPATIBLE = "xmlns:notXSLTCompatible";
```

Comment:
Shall we rename this to "XSLT_COMPATIBLE" for readability and change logics accordingly?